examples/hpc-slurm-v6-tpu.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hpc-slurm-v6
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hpc-slurm-v6
+  region: us-central1
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
+  # as a prefix. To refer to a local resource, prefix with ./, ../ or /
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - id: slurm_cluster
+    source: community/modules/scheduler/schedmd-slurm-gcp-v6-cluster
+    settings:
+      slurm_cluster_name: cluster1
+      partitions: [{
+        partition_conf: {
+          Default = "YES"
+        },
+        partition_name: "debug",
+        partition_nodeset_tpu: ["v2x8"],
+      }]
+      nodeset_tpu: [{
+        nodeset_name: "v2x8",
+        node_type: "v2-8",
+        tf_version: "2.9.1",
+        preemptible: true,
+        zone: "us-central1-b",
+        preserve_tpu: true,
+        enable_public_ip: true,
+        node_count_dynamic_max: 5,
+        node_count_static: 0,
```

Comment:
IIUC, `node_count_static` is 0 by default, let's omit it.

Reply:
Done.